forte/pipeline.py

```diff
@@ -219,6 +219,9 @@ def __init__(
         # Indicate whether do type checking during pipeline initialization
         self._do_init_type_check: bool = do_init_type_check
 
+        # The version of intermediate representation format
+        self.FORTE_IR_VERSION: str = "0.0.1"
```

Comment:
Maybe it is better to put this string in this file "https://github.com/asyml/forte/blob/master/forte/version.py", and then refer here.